Development.md

```diff
@@ -0,0 +1,235 @@
+# XAI Infrastructure
+
+## Introduction
+
+The XAI infrastructure is built in [Sentry Node Monorepo xai-foundation/sentry](https://github.com/xai-foundation/sentry
+
+This monorepo holds the following packages:
+
+- Core package
+  - shared libraries like operatorRuntime and blockchain & subgraph interactions
+- custom ui libraries
+  - shared ui components for websites and desktop client
+- CLI
+- Desktop App
+- sentry.xai.games website
+- app.xai.games website
+- smart contracts
+- public node
+- subgraph sync
+
+## Development cycle
+
+Ongoing development will be done on the sentry fork https://github.com/xai-foundation/sentry-development (this requires write access on that repo).
+
+- Feature branches for epics will be created on the production repo (`xai-foundation/sentry`) (this requires write access on that repo).
+- Working branches will be created **from** `origin/xai-foundation/sentry/feature-branch` **in** `xai-foundation/sentry-development:working-branch`
+- PRs get created on `xai-foundation/sentry-development:working-branch` back into `xai-foundation/sentry/feature-branch`
+- Every PR links to its story and has a description on
+  - What has been done
+  - How has it been tested
+- Every story links to it's PR for it to be marked as finished
+- Once the PR is approved the PR requester will merge and deploy, mark the story as delivered and add info for testing to the story
+
+## Local development
+
+Currently the monorepo can be installed and built using **pnpm** except `web-staking` that has yet to be integrated and needs to be managed with **npm** (https://www.pivotaltracker.com/story/show/187887214)
+
+[NX](https://nx.dev/features/run-tasks) can be used for local script management, currently for every app except `web-staking`
+
+Run the different apps by their commands in the root's `package.json` like
+
+- `pnpm cli` for running the CLI
+- `pnpm web` for running sentry-xai.games locally
+- `pnpm desktop` for running the desktop app locally
+
+For local release building and signing SSL.com access is requires. Currently the credentials are set in the github repositories sentry & sentry-development, however, sentry.development is currently breaking on sign.
```

Comment:
What needs to happen to fix the breaking sign?

Reply:
We have a story to track that https://www.pivotaltracker.com/story/show/187983657, I will add the reference to the docs

I have extended this story to update the docs once this is done
https://www.pivotaltracker.com/story/show/188048972